gcp/README.md

```diff
@@ -449,6 +449,13 @@ See [Getting started: One-time Stackdriver Workspace setup](README.md#one-time-s
 1. Enter channel name including "#". Click "Test Connection" and then "Save".
 1. Now you can use new notification channel in `gcp-stackdriver-monitoring` module. Here is example json: `{"type":"slack","labels":{"channel_name":"#alerts"},"user_labels":{},"enabled":{"value":true},"immutable":{"value":true}}`.
 
+### Fixing errors related to the Stackdriver deployment
+
+If during a deployment any error is raised because either a log based metric or an alert already exist. It is possible to remove all the Stackdriver resources and let GPII-infra to recreate them.
+
+1. `rake clean_lbm_alerts`
```

Comment:
This now does not match the task name -  `clean_stackdriver_resource`.

Reply:
fixed